ansible_base/resource_registry/tasks/sync.py

```diff
@@ -317,8 +316,9 @@ def _cleanup_orphans(self, resource_type, manifest_list):
         if self.deleted_count:
             self.write(f"Deleting {self.deleted_count} orphaned resources")
             for orphan in resources_to_cleanup:
```

Comment:
If we skip some resources below, the number of deleted resources should = resources_to_cleanup.count() - number of skipped resources.